### PR TITLE
add email validator

### PR DESCRIPTION
**Pull request by Slashek**

### CODE REVIEW

File: app/views/pages/developer-guide/forms/validation.liquid [hunk 1]
```diff
@@ -74,6 +74,17 @@ password_confirmation:
...
 
+### email
+
+Checks if a provided value is a valid email address.
+
+```yaml
+author_email:
+  validation:
+    email: true
+    presence: true
```
Comment (pavelloz) [inline app/views/pages/developer-guide/forms/validation.liquid]: Is presence required to demonstrate it?

Comment (Slashek) [reply app/views/pages/developer-guide/forms/validation.liquid]: no, it's not - we can remove it, however, that's the most common scenario

Comment (pavelloz) [reply app/views/pages/developer-guide/forms/validation.liquid]: i would remove it then to not confuse reader

Comment (Slashek) [reply app/views/pages/developer-guide/forms/validation.liquid]:
```suggestion
```

**Pull request merged**